rpc_jobs/rpc_gating_lint.yml

```diff
@@ -18,6 +19,11 @@
         deleteDir()
         stage("Prepare"){
           git url: env.ghprbAuthorRepoGitUrl, branch: ghprbSourceBranch
+          /* Set mtime to a constant value as git doesn't track mtimes but
+           * docker 1.7 does, this causes cache invalidation when files are
+           * added
+           */
+          sh "touch -t 201704100000 *.txt"
```

Comment:
If we always set the mtime for the files, then we'll run into issues if we do actually update the files.